next/components/Molecules/BranchDetails/BranchDetails.tsx

```diff
@@ -50,12 +50,21 @@ const BranchDetails = ({ branch }: PageProps) => {
               </div>
             </div>
 
-            {/* TODO: Extract description from subbranches */}
             {body?.trim() ? (
               <div id="description">
-                <h3 className="text-h3">{t('description')}</h3>
-                <div className="pt-5 text-[16px] text-foreground-body">
+                <h2 className="text-h3">{t('description')}</h2>
+                <div className="flex flex-col gap-4 pt-5 text-[16px] text-foreground-body">
                   <RichText content={body} />
+                  {subBranches?.data.map((subBranch) => {
+                    const { body: subBranchBody, title: subBranchTitle } =
+                      subBranch.attributes ?? {}
+                    return subBranchBody && subBranchBody.trim() ? (
+                      <>
```

Comment:
You are missing unique `key` here. `<Fragment key={...}>`
Usually we use `id` of entity.

This eslint rule is broken somehow... :/